backend/app/services/adapters/task_kinds/queries.py

```
@@ -515,28 +493,24 @@ def get_user_tasks_by_title_with_pagination(
-        count_sql = text(
-            """
+        count_sql = text("""
             SELECT COUNT(*) FROM tasks
             WHERE user_id = :user_id
             AND kind = 'Task'
             AND is_active = true
             AND namespace != 'system'
-        """
-        )
+        """)
         total_result = db.execute(count_sql, {"user_id": user_id}).scalar()
```

Ruff (0.14.14)

[error] Local variable `total_result` is assigned to but never used

Remove assignment to unused variable `total_result`

(F841)